gems/smithy/spec/fixtures/service_index/model.smithy

```diff
@@ -68,13 +72,38 @@ operation NestedResourceOperation {
     input: Structure
 }
 
+operation MapOperation {
+    input:= {
+        map: StructureMap
+    }
+}
+
 @documentation("This is a structure shape.")
 structure Structure {
     @required
     id: String
     name: String
 }
 
+list StructureList {
+    member: StructureMember
+}
+
```

Comment:
ListMember? And could also target string

Reply:
It can't that won't test the case.  It needs to be a shape that is not referenced anywhere else.

Reply:
I was saying I think the member should be called ListMember, that targets a string?